utils/deprecations.py

```diff
@@ -0,0 +1,12 @@
+import logging
+from os.path import basename
+
+log = logging.getLogger(__name__)
+
+
+def deprecate_old_command_line_tools(file_name):
+    name = basename(file_name)
+    if name == 'dd-forwarder' or name == 'dogstatsd' or name == 'dd-agent':
```

Comment:
These match the command names (`basename(sys.argv[0])`) but not the file names (`basename(__file__)`) ?